backend/package.json

```
@@ -0,0 +1,45 @@
+{
+  "name": "minimalistic-calculator-calories",
+  "version": "1.0.0",
+  "description": "Backend минималистичного калькулятора калорий",
+  "author": "Евгения Никонова <begale.elgaeb@ya.ru>",
+  "repository": "https://github.com/beagle-elgaeb/pet-clculator-calories.git",
+  "main": "src/app.ts",
+  "scripts": {
+    "start": "node ./build/app.js",
+    "dev": "ts-node-dev ./src/app.ts",
+    "build": "tsc"
+  },
+  "dependencies": {
+    "@types/pg": "^8.6.4",
+    "bcryptjs": "^2.4.3",
+    "celebrate": "^15.0.0",
+    "cookie-parser": "^1.4.6",
+    "cors": "^2.8.5",
+    "dotenv": "^10.0.0",
+    "express": "^4.17.1",
+    "express-rate-limit": "^5.5.1",
+    "express-winston": "^4.2.0",
+    "helmet": "^4.6.0",
+    "jsonwebtoken": "^8.5.1",
+    "knex": "^0.95.15",
+    "mongoose": "^6.0.13",
+    "objection": "^3.0.1",
+    "pg": "^8.7.1",
+    "validator": "^13.7.0",
+    "winston": "^3.3.3"
```

удалить mongoose, validator
```suggestion
    "knex": "^0.95.15",
    "objection": "^3.0.1",
    "pg": "^8.7.1",
    "winston": "^3.3.3"
```